pom.xml

```diff
@@ -582,6 +582,12 @@
         <scope>compile</scope>
         <optional>true</optional>
       </dependency>
+      <!-- OS Detection for loading the native libraries -->
+      <dependency>
+        <groupId>kr.motd.maven</groupId>
+        <artifactId>os-maven-plugin</artifactId>
+        <version>1.2.3.Final</version>
+      </dependency>
```

Comment:
@trustin if you disagree, i can make your proposed change (or you can feel free to push to this branch if GHE allows that?) Just let me know.